.circleci/config.yml

```diff
@@ -12,8 +12,6 @@ jobs:
           MYSQL_DATABASE: ctldb
           MYSQL_USER: ctldb
           MYSQL_PASSWORD: ctldbpw
-
-    resource_class: large
```

Comment:
did you mean to bring in this change?

Reply:
This was from before #30, when I was trying to figure out why CI wasn't running :) It's no longer needed!